packages/cel/src/index.ts

```diff
@@ -64,22 +56,3 @@ export function createEnv(namespace: string, registry: Registry): CelEnv {
   });
   return env;
 }
-
-function throwIfError<T>(result: CelError | T): T {
-  if (result instanceof CelError) {
-    throw result;
-  }
-  return result;
-}
-
-export function newDuration(seconds: bigint, nanos: number): Duration {
-  return throwIfError(_newDuration(0, seconds, nanos));
-}
-
-export function parseDuration(duration: string): Duration {
-  return throwIfError(_parseDuration(0, duration));
-}
-
-export function newTimestamp(seconds: bigint, nanos: number): Timestamp {
-  return throwIfError(_newTimestamp(0, seconds, nanos));
-}
```

Comment:
I was not sure why we want to export these

Reply:
👍  `@bufbuild/protobuf/wkt` has constructor functions. If we think they aren't sufficient, we should improve them, not create an alternative here.

Reply:
To clarify: It's fine that we have the _internal_ function `createTimestamp` - agree that it should not be exported.